src/scraper.rs

```diff
@@ -184,15 +184,14 @@ impl Scraper {
             .into_iter()
             .filter(|candidate| Scraper::should_visit(candidate, &url))
             .for_each(|next_url| {
-                let next_full_url = url.join(&next_url).unwrap();
+                let mut next_full_url = url.join(&next_url).unwrap();
+                next_full_url.set_fragment(None);
```

Comment:
If you do this, you will not download the correct webpage. For example: ``http://xxxx.xxx/test?name=foo`` prints ``foo.png`` and ``http://xxxx.xxx/test?name=bar`` prints ``bar.png``. This will attempt to download ``http://xxxx.xxx/test`` and the content will not be correct

Reply:
Huuu. That already what we are doing, sorry. Maybe we should just never do a ``.set_fragment(None)``

Reply:
You're talking about query strings, fragments are `#` anchors on the page.

Reply:
Ohh ok, my bad